march_joint_inertia_controller/package.xml

```diff
@@ -1,21 +1,32 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
-  <name>march_joint_inertia_controller</name>
-  <version>0.0.0</version>
-  <description>ROS controller which can be customized to operate in different circumstances.  </description>
+    <name>march_joint_inertia_controller</name>
+    <version>0.0.0</version>
+    <description>ROS controller which can be customized to operate in different circumstances.</description>
```

Comment:
Why the different spacing in this entire file?

Reply:
probably because of: Cntrl+Alt+L

Reply:
Cntrl+Alt+L